lib/elixir/lib/calendar.ex

```diff
@@ -194,6 +234,16 @@ defmodule Date do
   end
 
   @doc """
+  Returns the number of days in a week.
+
+  Always returns 7 days in a week irrespective of the
+  calendar in use.
+  """
+  def days_in_a_week() do
```

Comment:
Also did you mean to suggest `{RD, milliseconds}` or `{RD, microseconds}`?.  `Elixir.Time` is microseconds resolution so I assume that would be the intent for consistency?